regparser/layer/formatting.py

```diff
@@ -236,7 +236,7 @@ def match_data(self, match):
 
 
 class Footnotes(PlaintextFormatData):
-    """E.g.     [^4](Contents of footnote with escaped parens: \(\)s)"""
+    r"""E.g.     [^4](Contents of footnote with escaped parens: \(\)s)"""
```

Comment:
The `r` prefix just means "raw", so we don't need to double-escape the slashes. If we don't use an `r`, we're technically encoding bad escape sequences (`\(` and `\)`). What should we do? I'd like to include the indication that parens can be escaped, but not if it makes the code harder to understand.